experiments/AMIP/modular/coupler_driver_modular.jl

```diff
@@ -407,14 +426,26 @@ step!(land_sim, Δt_cpl)
 step!(ocean_sim, Δt_cpl)
 step!(ice_sim, Δt_cpl)
 
-# 3) coupler re-imports updated surface fields and calculates turbulent fluxes on the atmos grid, while updating atmos sfc_conditions
-import_combined_surface_fields!(cs.fields, cs.model_sims, cs.boundary_space, turbulent_fluxes) # i.e. T_sfc, albedo, z0, beta
-compute_combined_turbulent_fluxes!(cs.model_sims, cs.fields, turbulent_fluxes)
+# 3) coupler re-imports updated surface fields and calculates turbulent fluxes, while updating atmos sfc_conditions
```

Comment:
Maybe add a comment here explaining the difference between calculations for each turbulent flux type

Reply:
Included in comments inside the if.. else.. loop below. Lmk if you feel like it needs to be clarified further.